mbq/client/contrib/permissions.py

```diff
@@ -427,3 +448,18 @@ def has_all_permissions(  # noqa: F811
             result=result,
         )
         return result
+
+    def get_org_refs_for_permission(
+        self,
+        person_id: UUIDType,
+        scope: str,
+    ) -> Iterable[str]:
+        """ Given a person and permission scope return all of the org or
+        location references where the person has that permission.
+        """
+        result = self.os_core_client.fetch_org_refs_for_permission(person_id, scope)
+        self.collector.increment(
+            "org_refs_for_permission",
+            tags={"call": "org_refs_for_permission", "result": str(result), "scope": scope},
+        )
```

Comment:
can you please do a 

```self.registrar.emit("event_name_of_your_choice", person_id, scope, result=result) ```

:)

Reply:
I did not see those before. Adding them now